### PR TITLE
combo 4.2.5 + text 2.3.0

### PR DESCRIPTION
**Pull request by ujoni**
Needs to be cherry-picked to 2.1 branch.

Release as 2.1.1?

---
This change is [Reviewable](https://reviewable.io/reviews/vaadin/vaadin-combo-box-flow/207)

### CODE REVIEW

File: vaadin-combo-box-flow/pom.xml [hunk 1]
```diff
@@ -46,13 +46,13 @@
         <dependency>
             <groupId>org.webjars.bowergithub.vaadin</groupId>
             <artifactId>vaadin-combo-box</artifactId>
-            <version>4.2.0</version>
+            <version>4.2.2</version>
```
Comment (web-padawan) [inline vaadin-combo-box-flow/pom.xml]: please include 4.2.4 released today, as it includes 2 fixes for Flow counterpart (for both V12 and V13)

Comment (mehdi-vaadin) [reply vaadin-combo-box-flow/pom.xml]: When are you going to update `vaadin-combo-box` version to `4.2.4` in the `platform`? Based on the current version in the platform we haven't used the latest version (`4.2.4`).

Comment (web-padawan) [reply vaadin-combo-box-flow/pom.xml]: I see. Will submit a PR to the platform ASAP.

Comment (mehdi-vaadin) [reply vaadin-combo-box-flow/pom.xml]: Great. Thank you. We will update the version to 4.2.4 then.

Comment (ujoni) [reply vaadin-combo-box-flow/pom.xml]: Ah, I thought there were some problems with it. I must have misunderstood.

**Pull request merged**